integrations/github/highlights.scm

```diff
@@ -0,0 +1,214 @@
+; ; Highlighting for TLA+ files on GitHub
+; ; Capture names found here:
+; ; https://github.com/tree-sitter/tree-sitter/blob/f5d1c0b8609f8697861eab352ead44916c068c74/cli/src/highlight.rs#L150-L171
+
```

Comment:
I was unable to find any docs on the GitHub integration. Does it support `locals.scm`?

Reply:
I just merged github highlighting with our standard query highlighting so if github supports local/reference highlighting, it'll be used.